charts/apisix/templates/pdb.yaml

```diff
@@ -15,8 +15,11 @@
 # limitations under the License.
 
 {{- if (.Values.podDisruptionBudget.enabled) }}
+{{ if semverCompare "<1.21-0" .Capabilities.KubeVersion.Version -}}
 apiVersion: policy/v1beta1
+{{- else -}}
 kind: PodDisruptionBudget
+{{- end }}
```

Comment:
Did you mean to specify apiVersion policy/v1 here? I dont think the logic is quite right.

Reply:
https://github.com/apache/apisix-helm-chart/pull/719#pullrequestreview-1948146765

Reply:
Sorry, I will update it.